features/step_definitions/payment_steps.rb

```diff
@@ -28,7 +28,7 @@
   Capybara::Screenshot.screenshot_and_open_image
 end
 
-Then(/^I should see "([^"]*)" on the order confirmation$/) do |content|
-  sleep(4)
+Then(/^I should see "([^"]*)" on the order confirmation page$/) do |content|
+  sleep(2)
```

Comment:
Right, there is already a sleep right after clicking the button. Will change it asap and push it!